### DISCUSSION TITLE
Feature suspendable version of useTracker hook for async reactive functions, and also useFind / useSubscribe

### DISCUSSION THREAD

**Discussion by bratelefant** [TheRealNate/react-native-meteor#127] [Ideas]
Hi all,

I thought it could be maybe a nice (especially looking towards meteor 3.0) to have suspendable version of the `useTracker` hook, similar to [the non-native version](https://github.com/meteor/react-packages/blob/master/packages/react-meteor-data/README.md#suspendable-version-of-hooks).

Same for `useFind` and `useSubscribe`.

**Comment by jankapunkt** [comment]
Yes, totally agree. In general we should update the tracker to work as close as the "original" one.
We could already work on that, once we know that this topic is 100% implement , even if Meteor 3.0 has not been in beta/rc phase yet. I will ask the team about it.
